src/ucp/api/ucp.h

```diff
@@ -1189,6 +1227,31 @@ unsigned ucp_worker_progress(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_WORKER
+ * @brief Poll for endpoints with ready to consume streaming data.
+ *
+ * This non-blocking routine returns endpoints on a worker which have ready
+ * to consume streaming data. The ready endpoints are placed in @poll_eps array,
+ * and the function return value indicates how many are there.
+ *
+ * @param [in]   worker    Worker to poll.
+ * @param [out]  poll_eps  Pointer to array of endpoints, should be
+ *                         allocated by user.
+ * @param [in]   max_eps   Maximal number of endpoints which should be filled
+ *                         in @a poll_eps.
+ * @param [in]   flags     Reserved for future use.
+ *
+ * @return Negative value indicates an error according to @ref ucp_status_t.
+ *         On success, non-negative value (less or equal @a n) indicates actual
+ *         number of endpoints filled in @a eps array.
```

Comment:
I think it is clearer if you say less than or equal to 'max_eps', rather than to 'n'.
Similarly, why not say 'poll_eps array' rather than just 'eps array'?